src/ARMInterpreter.cpp

```diff
@@ -278,6 +278,20 @@ void A_SVC(ARM* cpu)
 
 void T_SVC(ARM* cpu)
 {
+    // Print from game. Execute `svc 0xFC` with the null-terminated string address in `r0`.
+    if ((cpu->CurInstr & 0xFF) == 0xFC)
+    {
+        u32 addr = cpu->R[0];
+        for (;;)
+        {
+            u32 c;
+            cpu->DataRead8(addr++, &c);
+            if (!c) break;
+            printf("%c", c);
```

Comment:
Please use `Platform::Log` instead of `printf`. Then the output can go wherever the frontend wants it to instead of always `stdout`.

Reply:
done

Reply:
Great, thank you.